src/Contracts/MASA.Scheduler.Contracts.Server/Requests/PaginationRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Scheduler.Contracts.Server.Requests;
+
+public class PaginationRequest : BaseRequest
```

Comment:
delete BaseRequest in project and add  Masa.BuildingBlocks.Service.Contracts replace